src/System.Runtime.Extensions/src/System/IO/Path.Windows.cs

```diff
@@ -125,7 +125,25 @@ public static bool IsPathRooted(string path)
                 PathInternal.CheckInvalidPathChars(path);
 
                 int length = path.Length;
-                if ((length >= 1 && PathInternal.IsDirectorySeparator(path[0])) || 
+                if ((length >= 1 && PathInternal.IsDirectorySeparator(path[0])) ||
+                    (length >= 2 && path[1] == VolumeSeparatorChar))
+                    return true;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Tests if the given path starts with a root that is not equal to the 
+        /// forward slash.
+        /// </summary>
+        public static bool IsNonSeparatorRooted(string path)
+        {
+            if (path != null)
+            {
+                PathInternal.CheckInvalidPathChars(path);
+
+                int length = path.Length;
+                if ((length >= 1 && (path[0] == '\\')) ||
```

Comment:
It is better to be consistent I think.  It will be mighty confusing as people expect `AltDirectorySeparator == DirectorySeparator` on Windows.

If you do change just the single slash case you'll also be changing what `//./` and `//?/` do, which I'm not sure is a good thing.